curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-024.md

```diff
@@ -1,42 +1,70 @@
 ---
-id: 6193ed97ae313e4331c9078a
+id: 61fd990577d8227dd93fbeeb
 title: Step 24
 challengeType: 0
 dashedName: step-24
 ---
 
 # --description--
 
-Give the `span` elements within your second `.row total` element the following text, in order: `Total`, `$750`, `$600`, and `$475`.
+In your fourth `tr` element, add a `th` element with the text `Total Liabilities`. Wrap the text `Liabilities` in a `span` element with the `class` attribute set to `sr-only`.
+
+Following that, add three `td` elements with the following text (in order): `$550`, `$750`, `$500`. Give the third `td` element a `class` attribute set to `current`.
```

Comment:
```suggestion
Following that, add three `td` elements with the following text (in order): `$750`, `$600`, `$475`. Give the third `td` element a `class` attribute set to `current`.
```

The numbers are little off in the description. I think my suggestion is accurate.